src/main/java/org/apache/sling/distribution/journal/impl/subscriber/DistributionSubscriber.java

```diff
@@ -125,6 +129,12 @@ public class DistributionSubscriber {
     @Reference
     private SubscriberReadyStore subscriberReadyStore;
 
+    @Reference(name = "packageHandlerFactory")
```

Comment:
you don't need the `name` here, but you should specify a default target filter `(name=default)`

Reply:
If we ditch the packageHandlerFactory, the reference would become
```
@Reference(name = "packageHandler")
```

Reply:
this name will be used in the EDS subscriber configurer to set the package handler factory to the EDS specific one:
```
props.put("packageHandlerFactory.target", "(name=edge-delivery)");
```